src/rdb/rdb.c

```diff
@@ -411,6 +411,11 @@ rdb_stop(struct rdb *db)
 {
 	bool deleted;
 
+	if (db == NULL) {
```

Comment:
As we discussed, this shouldn't be necessary, but I'm fine with landing it with the workaround for now and I'll figure out how to avoid calling this in the non-db case.